### PR TITLE
Tenant vault deployment data

### PR DESCRIPTION
**Pull request by gjoranv**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by tokle — DISMISSED** [review]

File: configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionSerializer.java [hunk 1]
```diff
@@ -54,6 +56,7 @@ void write(SessionZooKeeperClient zooKeeperClient, ApplicationId applicationId,
                                                              dockerImageRepository,
                                                              athenzDomain,
                                                              quota,
+                                                             tenantVaults,
```
Comment (hmusum) [inline configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionSerializer.java:59]: You need to call `zookeeperClient.writeTenantVaults()` before `if (writeSessionData.value())`
as well, this code is only called if a feature flag is set. This code should be cleaned up and explained more I see, I'll look into that

**Review by hmusum — CHANGES_REQUESTED** [review]
Generally looks great, need one change

File: configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionSerializer.java (hunk 1 above)
Comment (gjoranv) [reply configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionSerializer.java:60]: Thanks! Missed that one.

**Comment by gjoranv** [comment]
@hmusum PTAL

**Review by hmusum — APPROVED** [review]
👍

**Pull request merged**